src/components/Board.js

```diff
@@ -51,7 +49,14 @@ const csvHeaders = [
 class Board extends React.Component {
   constructor() {
     super();
-    this.state = { totalData: [], totalContributions: 0, totalDistance: 0, query: '', isLoading: true };
+    this.state = {
+      totalData: [],
+      totalContributions: 0,
+      totalDistance: 0,
+      query: '',
+      isSearcAtStart: false,
```

Comment:
Typo, missing an H. Maybe we could find a better name. @bencpeters ideas?

Reply:
I think I'd just called it `startsWithSearch` (and `toggleStartsWithSearch`)